nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -71,7 +76,7 @@ extide/gradle/external/gradle-7.3-bin.zip ide/c.jcraft.jzlib/external/jzlib-1.1.
 extide/gradle/external/gradle-7.3-bin.zip ide/libs.antlr4.runtime/external/antlr4-runtime-4.7.2.jar
 extide/gradle/external/gradle-7.3-bin.zip ide/o.apache.commons.lang/external/commons-lang-2.6.jar
 extide/gradle/external/gradle-7.3-bin.zip ide/o.eclipse.jgit/external/org.eclipse.jgit-5.7.0.202003110725-r.jar
-extide/gradle/external/gradle-7.3-bin.zip java/maven.embedder/external/apache-maven-3.6.3-bin.zip
+extide/gradle/external/gradle-7.3-bin.zip java/maven.embedder/external/apache-maven-3.8.4-bin.zip
```

Comment:
Is this really needed?

Reply:
removing this line make more warnings. So I guess this is needed

Reply:
I tested myself and the build system makes a deep inspection and finds overlaps. gradle and maven both contain commons-io and more libraries.